theme/views.py

```diff
@@ -4,6 +4,7 @@
 from django.views.generic import TemplateView
 from django.http import HttpResponse
 from django.shortcuts import redirect
+from django.conf import settings
```

Comment:
It seems so although I have not had a chance to test it out. See https://github.com/stephenmcd/mezzanine/blob/master/mezzanine/conf/__init__.py for details, where the top comment says the following:
Drop-in replacement for ``django.conf.settings`` that provides a consistent access method for settings defined in applications, the project or Django itself. Settings can also be made editable via the admin.